### ISSUE TITLE
Include firmware.hex with PyPI package

### ISSUE THREAD

**Issue by Lukasa** [ntoll/microrepl#2]
It seems like it might be particularly useful to have firmware.hex also distributed with the .tar.gz sdist from PyPI.

**Comment by ntoll** [comment]
Agreed. You should also be able to automagically flash the device if a micro:bit is found but isn't responding to the urepl. ;-)
